### DISCUSSION TITLE
Pure Dart

### DISCUSSION THREAD

**Discussion by MaximilianFlechtner** [isar/isar#188] [Help]
How can I use the database in pure dart. For example in [Alfred](https://github.com/rknell/alfred)

**Comment by simc** [comment]
You certainly can. But you need to put the Isar Core binaries at the correct location or initialize it https://github.com/isar/isar/blob/7e64a761529f6ab2ad0aaf432ec8a15baf094a93/packages/isar/lib/src/native/isar_core.dart#L44-L56

There are no docs yet how to do it. Let me know if you need help.

**Comment by guyluz11** [comment]
Add this line at the beginning of your code

```dart
await Isar.initializeIsarCore(download: true)
```
